packages/components/src/TitleBlockZen/TitleBlockZen.module.scss

```diff
@@ -39,6 +39,20 @@
       background-color: $dt-color-background-color-admin;
     }
 
+    @media (min-width: $layout-breakpoints-large) {
+      &.homeVariant {
+        background-color: transparent;
+
+        h1 {
```

Comment:
Generally we avoid use tags directly unless absolutely necessary. This is mainly as classes are a quicker reference tool within our code base and it helps us to avoid cases where a tag might change. In these instances it can be confusing to find the diff as we often don't use tag directly, ie: we use the `heading-1` variant vs an `<h1>` tag.

Would probably be the `titleTextOverride` class instead?